Add TA-88047 (#15178)
cockroachdb/docs: advisories/a88407.md

---
title: Technical Advisory 88047
advisory: A-88047

> summary: Querying a <code>REGIONAL BY ROW</code> or partitioned multi-region table could produce incorrect results if the query had a <code>LIMIT</code> of less than 100,000 and used the inverted index.

Reviewer: ```suggestion
summary: Querying a <code>REGIONAL BY ROW</code> or partitioned multi-region table could produce incorrect results if the query has a <code>LIMIT</code> of less than 100,000 and uses an inverted index.
```

Author: Fixed!

toc: true
affected_versions: v22.1.0 to v22.1.7
advisory_date: 2022-09-29
docs_area: releases
---

Publication date: {{ page.advisory_date | date: "%B %e, %Y" }}

## Description

> In CockroachDB [v22.1.0](../releases/v22.1.html#v22-1-0) to [v22.1.7](../releases/v22.1.html#v22-1-7), querying a [`REGIONAL BY ROW`](../stable/multiregion-overview.html#regional-by-row-tables) or [partitioned](../stable/partitioning.html) multi-region table with an [inverted index](../stable/inverted-indexes.html) could produce incorrect results if the query had a [`LIMIT`](../stable/limit-offset.html) of less than 100,000 and used the inverted index. In this case, it was possible that the [optimizer](../stable/cost-based-optimizer.html) would erroneously discard the predicate over the inverted index and therefore return some rows that did not match the predicate.

Reviewer: ```suggestion
In CockroachDB [v22.1.0](../releases/v22.1.html#v22-1-0) to [v22.1.7](../releases/v22.1.html#v22-1-7), querying a [`REGIONAL BY ROW`](../stable/multiregion-overview.html#regional-by-row-tables) or [partitioned](../stable/partitioning.html) multi-region table with an [inverted index](../stable/inverted-indexes.html) could produce incorrect results if the query has a [`LIMIT`](../stable/limit-offset.html) of less than 100,000 and uses an inverted index. In this case, it is possible that the [optimizer](../stable/cost-based-optimizer.html) will erroneously discard the predicate over the inverted index and therefore return some rows that do not match the predicate.
```

Author: Fixed!
